### DISCUSSION TITLE
Highlight row when clicked

### DISCUSSION THREAD

**Discussion by GaryLouw** [KevinVandy/material-react-table#404] [Q&A] [answered]
Hi, I would like to highlight a row when clicked and am not too sure how to do it

**Comment by KevinVandy** [comment, marked as answer]
Looking for the Manual Row Selection without checkboxes example? https://www.material-react-table.com/docs/guides/row-selection#manual-row-selection-without-checkboxes

**Reply by GaryLouw** [reply]
Exactly what I need, thank you a lot!!
